### PR TITLE
feat(cargo): default to update-lockfile

### PR DESCRIPTION
**Pull request by rarkins**
## Changes

Change Cargo's default behavior from rangeStrategy=bump to rangeStrategy=update-lockfile

## Context

Align with other managers

## Documentation (please check one with an [x])

- [x] I have updated the documentation, or
- [ ] No documentation update is required

## How I've tested my work (please select one)

I have verified these changes via:

- [ ] Code inspection only, or
- [x] Newly added/modified unit tests, or
- [ ] No unit tests but ran on a real repository, or
- [ ] Both unit tests + ran on a real repository

### CODE REVIEW

File: docs/usage/config-overview.md [hunk 1]
```diff
@@ -108,6 +108,8 @@ Read the [Self-hosted experimental environment variables](./self-hosted-experime
 Finally, there are some special environment variables that are loaded _before_ configuration parsing because they are used during logging initialization:
 
 - `LOG_CONTEXT`: a unique identifier used in each log message to track context
+- `LOG_FILE`: used to enable file logging and specify the log file path
```
Comment (HonkingGoose) [inline docs/usage/config-overview.md:111]: I guess there's something going wrong with your latest force-push. It looks like you're editing more files than intended?

**Review by yanns — APPROVED** [review]

**Review by viceice — DISMISSED** [review]

**Comment by rarkins** [comment]
I want to back port this to v37 as I think the "fix" I made to bump behavior is more unexpected/undesired than switching to update lock file

**Comment by viceice** [comment]
> I want to back port this to v37 as I think the "fix" I made to bump behavior is more unexpected/undesired than switching to update lock file

sounds good to me

**Comment by rarkins** [comment]
Replaced by #28826

**Pull request closed without merging**